### ISSUE TITLE
unzip does not support >65,535 files

### ISSUE THREAD

**Issue by neilrackett** [101arrowz/fflate#137]
**How to reproduce**

If you try to `unzip` or `unzipSync` a zip file containing >65,535 files, only 65,535 files will be listed.

For example:

```javascript
// Load zip file containing 100,000 files into ArrayBuffer
const zipArray = new Uint8Array(zipBuffer);
const files = unzipSync(zipArray);

console.log(Object.keys(files).length) // 65535
```

**The problem**

This issue occurs without any error messages, the number of files is simply truncated to 65,535.

**Comment by 101arrowz** [comment]
Could you provide a sample file (or how to generate one) so I can reproduce this?

**Comment by neilrackett** [comment]
No problem: here's a zip file containing 70,000 sequentially named text files I created using a simple node script:

```js
const fs = require('fs');
for (let i = 1; i <= 70000; i++) {
  fs.writeFileSync(`./files/file-${('000000' + i).substr(-6)}.txt`, 'x');
}
```

[70000-files.zip](https://github.com/101arrowz/fflate/files/9041838/70000-files.zip)

**Comment by 101arrowz** [comment]
`fflate` isn't looking at the Zip64 end-of-central-directory record so it can only use the number of files given in the standard 16-bit EOCD record. This should be fixable, I'll look into it.

**Comment by 101arrowz** [comment]
Fixed in v0.7.4. Sorry for the long delay!

**Comment by neilrackett** [comment]
Amazing, thank you!
